cli/boilerplate/angular-soho/src/index.html

```diff
@@ -9,11 +9,15 @@
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="icon" type="image/x-icon" href="favicon.ico">
    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
-   <link rel="stylesheet" id="stylesheet" href="assets/ids-enterprise/css/theme-new-light.css" type="text/css">
+   <link rel="stylesheet" id="stylesheet" href="assets/ids-enterprise/css/theme-new-light.min.css" type="text/css">
+   <script>
+      // https://github.com/infor-design/enterprise/tree/main/src/components/locale#code-example---using-minified-files
+      window["SohoConfig"] = { minifyCultures: true };
```

Comment:
Would it be possible to configure this in `boilerplate/angular-soho/src/app/app.module.ts` instead? In the `APP_INITIALIZER` where the locale is set.

Functionally there shouldn't be a difference, but it's nice to do the locale/culture config in a single place

Reply:
The property `Locale.minify` of infor-design/enterprise project  isn't "published" in `https://github.com/infor-design/enterprise-ng/blob/main/projects/ids-enterprise-typings/lib/locale/soho-locale.d.ts` to be directly used in Angular projects. I can add `Soho.Locale['minify'] = true;` to `app.module.ts`, but it looks a little bit tricky and isn't well "documented". How should we proceed @anhallbe? Should I create a PR in infor-design/enterprise-ng project to make the `Locale.minify` property available to Angular projects?

Reply:
You could add the following to extend the `SohoLocaleStatic` typing, for better type safety and documentation:

```typescript
declare global {
   interface SohoLocaleStatic {
      /**
       * NOTE: Not part of the official IDS typings. Since this is not part
       * of the official API, it may stop working at any time.
       */
      minify: boolean;
   }
}
```

and then in the APP_INITIALIZER:

```typescript
Soho.Locale.minify = true;
Soho.Locale.culturesPath = 'assets/ids-enterprise/js/cultures/';
return Soho.Locale.set('en-US').catch(err => Log.error('Failed to set IDS Locale', err));
```

Reply:
What do you think @tmcconechy, should the project @infor-design/enterprise-ng publish the `minify` property from @infor-design/enterprise projects `Locale`? I would avoid unofficial features in `odin new` generated files. The official documentation in @infor-design/enterprice only mentions the following:

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/35500068/199333737-7fd259cb-fb38-41b8-9934-2db4ebb98470.png">

That's the reason why I'm not sure how to handle this situation.